README.md

```diff
@@ -62,7 +62,10 @@ plugins: {
 			// Function called while the user is panning
 			onPan: function({chart}) { console.log(`I'm panning!!!`); },
 			// Function called once panning is completed
-			onPanComplete: function({chart}) { console.log(`I was panned!!!`); }
+			onPanComplete: function({chart}) { console.log(`I was panned!!!`); },
+			// Function called when pan fails because modifier key was not detected.
+			// event is the a hammer event that failed - see https://hammerjs.github.io/api#event-object
+			onModifierKeyFailed: function({chart, event}) { console.log(`I didn't start panning!`); }
```

Comment:
The event name is a bit surprising to me, how do you feel about `onPanRejectet`?